lib/publish_world_index_page.rb

```diff
@@ -0,0 +1,9 @@
+class PublishWorldIndexPage
```

Comment:
I do wonder here whether we might be able to combine some of these individual classes to publish single pages into a single thing that takes an argument of the presenter? Feels like we're adding a fair few of these now.

Reply:
Nice idea.  I'll add this as a separate commit.